plugins/devtools-backend/src/api/DevToolsBackendApi.ts

```diff
@@ -219,7 +219,12 @@ export class DevToolsBackendApi {
     const lockfilePath = paths.resolveTargetRoot('yarn.lock');
     const lockfile = await Lockfile.load(lockfilePath);
 
-    const deps = [...lockfile.keys()].filter(n => n.startsWith('@backstage/'));
+    const prefixes = this.config.getOptionalStringArray(
+      'devTools.packagePrefixes',
+    ) ?? ['@backstage/'];
```

Comment:
Would it make more sense to just use `@backstage/` instead of `@backstage`? I know that's how the code was already but now that I look at it again it seems not needed

Reply:
I thought about this a bit but might `@backstage` be too lenient given someone could have a `@backstage-complete-unreleated/package` that would match a `@backstage` prefix but not a `@backstage/` prefix. Maybe this doesn't matter at all?

Reply:
I've gone with the simpler `@backstage` prefix.